Fix TestCustomGateway integration tests on OCP

sridhargaddam opened the pull request:
The test case was not configuring the platform details in the Helm values, because of which the test was failing. This PR fixes it.

Fixes: https://github.com/istio/istio/issues/52973

istio-testing commented:
@sridhargaddam: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
integ-basic-arm64_istio | e0c52dcdda8abbaeb67ceed5357ce232e719ab68 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1830883436517134336) | true | `/test integ-basic-arm64`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

sridhargaddam commented:
/test integ-basic-arm64

istio-testing commented:
@sridhargaddam: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
integ-basic-arm64_istio | e0c52dcdda8abbaeb67ceed5357ce232e719ab68 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1830892004884615168) | true | `/test integ-basic-arm64`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

sridhargaddam commented:
The arm test is flaky and the following issue is reported to track it - https://github.com/istio/istio/issues/52965

dgn commented:
/test integ-basic-arm64

istio-testing commented:
@sridhargaddam: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
integ-basic-arm64_istio | e0c52dcdda8abbaeb67ceed5357ce232e719ab68 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1830916641454559232) | true | `/test integ-basic-arm64`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

dgn commented on tests/integration/pilot/ingress_test.go:
```diff
@@ -728,6 +733,9 @@ resources:
     memory: 40Mi
 `, rev)), 0o644)
+				if t.Settings().OpenShift {
+					_ = file.AppendToFile([]byte(`platform: openshift`), d)
```
as we're writing the file in the line above - wouldn't it be more efficient to append to the string before writing?

sridhargaddam replied:
I considered that, but I felt that using `file.AppendToFile` was cleaner and easier to read, so I chose this approach. I'm open to changing it if that's preferred.

howardjohn reviewed (commented):
does this test even make sense? `manifests/charts/gateways/istio-ingress` is not a supported chart to install with helm (`manifests/charts/gateway` is)

NancyR17 commented:
To resolve the issue, you need to ensure that your pod’s runAsUser value is within the acceptable ranges defined by the SCCs in your OpenShift cluster. This involves reviewing, creating, or modifying SCCs and ensuring they are correctly applied to the service account used by your Helm deployment.

dhawton commented:
/test integ-basic-arm64

sridhargaddam commented:
> does this test even make sense? `manifests/charts/gateways/istio-ingress` is not a supported chart to install with helm (`manifests/charts/gateway` is)

Thanks for reviewing @howardjohn. Updated the PR and removed the test case.

sridhargaddam commented:
/test integ-basic-arm64

keithmattix commented on tests/integration/pilot/ingress_test.go:
```diff
@@ -637,79 +638,6 @@ spec:
-			// TODO we could add istioctl as well, but the framework adds a bunch of stuff beyond just `istioctl install`
-			// that mess with certs, multicluster, etc
-			t.NewSubTest("helm").Run(func(t framework.TestContext) {
-				gatewayNs := namespace.NewOrFail(t, namespace.Config{Prefix: "custom-gateway-helm", Inject: inject})
```
Why are we removing a test?

sridhargaddam replied:
I removed it based on the review comments - https://github.com/istio/istio/pull/52974#pullrequestreview-2277956693

istio-testing commented:
@sridhargaddam: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 4c49f50485798d4145fc9bf8ae48e80c9a28c49c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/lint_istio/1832129654027194368) | true | `/test lint`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@sridhargaddam: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 4c49f50485798d4145fc9bf8ae48e80c9a28c49c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/lint_istio/1832129654027194368) | true | `/test lint`
integ-basic-arm64_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1832132205539758080) | true | `/test integ-basic-arm64`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@sridhargaddam: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 4c49f50485798d4145fc9bf8ae48e80c9a28c49c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/lint_istio/1832129654027194368) | true | `/test lint`
integ-basic-arm64_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1832132205539758080) | true | `/test integ-basic-arm64`
integ-distroless_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-distroless_istio/1832132208891006976) | true | `/test integ-distroless`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@sridhargaddam: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 4c49f50485798d4145fc9bf8ae48e80c9a28c49c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/lint_istio/1832129654027194368) | true | `/test lint`
integ-basic-arm64_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1832132205539758080) | true | `/test integ-basic-arm64`
integ-distroless_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-distroless_istio/1832132208891006976) | true | `/test integ-distroless`
integ-pilot_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot_istio/1832132221465530368) | true | `/test integ-pilot`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@sridhargaddam: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 4c49f50485798d4145fc9bf8ae48e80c9a28c49c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/lint_istio/1832129654027194368) | true | `/test lint`
integ-basic-arm64_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1832132205539758080) | true | `/test integ-basic-arm64`
integ-distroless_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-distroless_istio/1832132208891006976) | true | `/test integ-distroless`
integ-pilot_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot_istio/1832132221465530368) | true | `/test integ-pilot`
integ-ds_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-ds_istio/1832132210564534272) | true | `/test integ-ds`
integ-cni_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-cni_istio/1832132206361841664) | true | `/test integ-cni`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@sridhargaddam: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 4c49f50485798d4145fc9bf8ae48e80c9a28c49c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/lint_istio/1832129654027194368) | true | `/test lint`
integ-basic-arm64_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1832132205539758080) | true | `/test integ-basic-arm64`
integ-distroless_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-distroless_istio/1832132208891006976) | true | `/test integ-distroless`
integ-pilot_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot_istio/1832132221465530368) | true | `/test integ-pilot`
integ-ds_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-ds_istio/1832132210564534272) | true | `/test integ-ds`
integ-cni_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-cni_istio/1832132206361841664) | true | `/test integ-cni`
integ-ipv6_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-ipv6_istio/1832132214880473088) | true | `/test integ-ipv6`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@sridhargaddam: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 4c49f50485798d4145fc9bf8ae48e80c9a28c49c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/lint_istio/1832129654027194368) | true | `/test lint`
integ-basic-arm64_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1832132205539758080) | true | `/test integ-basic-arm64`
integ-distroless_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-distroless_istio/1832132208891006976) | true | `/test integ-distroless`
integ-pilot_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot_istio/1832132221465530368) | true | `/test integ-pilot`
integ-ds_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-ds_istio/1832132210564534272) | true | `/test integ-ds`
integ-cni_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-cni_istio/1832132206361841664) | true | `/test integ-cni`
integ-ipv6_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-ipv6_istio/1832132214880473088) | true | `/test integ-ipv6`
integ-pilot-multicluster_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot-multicluster_istio/1832132219796197376) | true | `/test integ-pilot-multicluster`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@sridhargaddam: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 4c49f50485798d4145fc9bf8ae48e80c9a28c49c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/lint_istio/1832129654027194368) | true | `/test lint`
integ-basic-arm64_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1832132205539758080) | true | `/test integ-basic-arm64`
integ-distroless_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-distroless_istio/1832132208891006976) | true | `/test integ-distroless`
integ-pilot_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot_istio/1832132221465530368) | true | `/test integ-pilot`
integ-ds_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-ds_istio/1832132210564534272) | true | `/test integ-ds`
integ-cni_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-cni_istio/1832132206361841664) | true | `/test integ-cni`
integ-ipv6_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-ipv6_istio/1832132214880473088) | true | `/test integ-ipv6`
integ-pilot-multicluster_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot-multicluster_istio/1832132219796197376) | true | `/test integ-pilot-multicluster`
integ-pilot-istiodremote-mc_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot-istiodremote-mc_istio/1832132217292197888) | true | `/test integ-pilot-istiodremote-mc`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@sridhargaddam: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
lint_istio | 4c49f50485798d4145fc9bf8ae48e80c9a28c49c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/lint_istio/1832129654027194368) | true | `/test lint`
integ-basic-arm64_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1832132205539758080) | true | `/test integ-basic-arm64`
integ-distroless_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-distroless_istio/1832132208891006976) | true | `/test integ-distroless`
integ-pilot_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot_istio/1832132221465530368) | true | `/test integ-pilot`
integ-ds_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-ds_istio/1832132210564534272) | true | `/test integ-ds`
integ-cni_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-cni_istio/1832132206361841664) | true | `/test integ-cni`
integ-ipv6_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-ipv6_istio/1832132214880473088) | true | `/test integ-ipv6`
integ-pilot-multicluster_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot-multicluster_istio/1832132219796197376) | true | `/test integ-pilot-multicluster`
integ-pilot-istiodremote-mc_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot-istiodremote-mc_istio/1832132217292197888) | true | `/test integ-pilot-istiodremote-mc`
integ-pilot-istiodremote_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot-istiodremote_istio/1832132218537906176) | true | `/test integ-pilot-istiodremote`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@sridhargaddam: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
integ-basic-arm64_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-basic-arm64_istio/1832132205539758080) | true | `/test integ-basic-arm64`
integ-distroless_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-distroless_istio/1832132208891006976) | true | `/test integ-distroless`
integ-pilot_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot_istio/1832132221465530368) | true | `/test integ-pilot`
integ-ds_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-ds_istio/1832132210564534272) | true | `/test integ-ds`
integ-cni_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-cni_istio/1832132206361841664) | true | `/test integ-cni`
integ-ipv6_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-ipv6_istio/1832132214880473088) | true | `/test integ-ipv6`
integ-pilot-multicluster_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot-multicluster_istio/1832132219796197376) | true | `/test integ-pilot-multicluster`
integ-pilot-istiodremote-mc_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot-istiodremote-mc_istio/1832132217292197888) | true | `/test integ-pilot-istiodremote-mc`
integ-pilot-istiodremote_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/integ-pilot-istiodremote_istio/1832132218537906176) | true | `/test integ-pilot-istiodremote`
lint_istio | e5a06198290a288b48a008674273339ce828859c | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/52974/lint_istio/1832132229862526976) | true | `/test lint`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes-sigs/prow](https://github.com/kubernetes-sigs/prow/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
In response to a cherrypick label: #52974 failed to apply on top of branch "release-1.23":
```
Applying: Fix TestCustomGateway integration tests on OCP
Applying: Remove TestCustomGateway/helm as it uses unsupported chart
Using index info to reconstruct a base tree...
M	tests/integration/pilot/ingress_test.go
Falling back to patching base and 3-way merge...
Auto-merging tests/integration/pilot/ingress_test.go
CONFLICT (content): Merge conflict in tests/integration/pilot/ingress_test.go
error: Failed to merge in the changes.
hint: Use 'git am --show-current-patch=diff' to see the failed patch
Patch failed at 0002 Remove TestCustomGateway/helm as it uses unsupported chart
When you have resolved this problem, run "git am --continue".
If you prefer to skip this patch, run "git am --skip" instead.
To restore the original branch and stop patching, run "git am --abort".

```

istio-testing commented:
In response to a cherrypick label: new issue created for failed cherrypick: #53088
